atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -521,7 +521,7 @@ private TransactionComponents createTransactionComponents(
                 metricsManager, lockAndTimestampServices, keyValueService);
         TransactionSchemaManager transactionSchemaManager = new TransactionSchemaManager(coordinationService);
 
-        TransactionService transactionService = initializeCloseable(() -> AtlasDbMetrics.instrument(
+        TransactionService transactionService = initializeCloseable(() -> AtlasDbMetrics.instrumentTimed(
```

Comment:
Hmm, we probably want this call at `SerializableTransactionManager:385`, not here. `TransactionService` is a separate class from `TransactionManager` (the former is the thing that manages start->commit timestamp mappings, the latter is the interface users have to AtlasDB).

Reply:
Ah yes, what was I thinking.

Reply:
Fixed